$OpenBSD: patch-ghc_mk,v 1.5 2011/04/25 22:06:43 kili Exp $

Include utf8-string and xhtml for now, because of dependencies.
Including *any* of those INTREE_ONLY_PACKAGES in the ghc packages
wasn't the brightest idea I had (see mtl library hell). The two
libraries left can and will be replaced by separate ports when
needed.

--- ghc.mk.orig	Sat Mar 26 19:10:06 2011
+++ ghc.mk	Mon Apr 25 17:21:44 2011
@@ -290,7 +290,7 @@ include rules/bindist.mk
 # They do not say "this package will be built"; see $(PACKAGES_xx) for that
 
 # Packages that are built but not installed
-INTREE_ONLY_PACKAGES := haskeline mtl terminfo utf8-string xhtml
+INTREE_ONLY_PACKAGES := haskeline mtl terminfo
 
 # Packages that, if present, must be built by the stage2 compiler,
 # because they use TH and/or annotations, or depend on other stage2
